### PR TITLE
Raj_Assignment_8_Done

### PR DESCRIPTION
**Pull request by RajAJun20**

### CODE REVIEW

File: src/rajAJun20/BankSystem.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+public class BankSystem {
+
+	int balance = 50000;
+	int creditCount = 0;
+	int debitCount = 0;
+	int printBalanceCount = 0;
+	static int creditCount1 = 0;
+	static int debitCount1 = 0;
+	static int printBalanceCount1 = 0;
```
Comment (KrishnaKTechnocredits) [inline src/rajAJun20/BankSystem.java]: Remove the initialization of all instance variables.

File: src/rajAJun20/BankSystem.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
+package rajAJun20;
+
+public class BankSystem {
+
+	int balance = 50000;
```
Comment (KrishnaKTechnocredits) [inline src/rajAJun20/BankSystem.java]: No Need to initialize this variable. perform credit() method first.

File: src/rajAJun20/BankSystem.java (hunk 1 above)
Comment (RajAJun20) [reply src/rajAJun20/BankSystem.java]: Changes done...!!

File: src/rajAJun20/BankSystem.java (hunk 2 above)
Comment (RajAJun20) [reply src/rajAJun20/BankSystem.java]: Changes done ....!!

**Review by KrishnaKTechnocredits — COMMENTED** [review]
Code looks good to me.

**Pull request merged**